<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Реєстрація</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img/img.qrc">
    <normaloff>:/icons/teacher.png</normaloff>:/icons/teacher.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 69, 71);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 180, 175);
border-radius: 20px;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sitka</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(32, 69, 71);</string>
             </property>
             <property name="text">
              <string>Реєстрація</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="messageLabel">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Посада</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="maxLength">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Інститут</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="instituteBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Кафедра</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="departmentBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>220</number>
             </property>
             <property name="topMargin">
              <number>150</number>
             </property>
             <item>
              <widget class="QPushButton" name="registerButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
               </property>
               <property name="text">
                <string>Зареєструватися</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="returnButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
               </property>
               <property name="text">
                <string>Назад</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
